app/views/static/index.html.slim

```diff
@@ -7,23 +7,29 @@
       header
         h1 OpenMensa - Die offene Mensa Datenbank.
       .content
+        = link_to about_path do
+          = image_tag image_path('compact.png'), id: 'compact', class: 'illustrate'
         p
           ' OpenMensa ist die kostenfreie Datenbank für Mensen und Cafeterien in Deutschland.
+          ' Wir stellen eine zentralle Austauschplatform zwischen Essensanbietern und Nutzern
+          ' bzw. deren Apps dar.
+          ' Hier findest du die Speisepläne zahlreicher Mensen und Cafés in Deutschland:
+        p.action
+          = link_to about_path, class: "btn btn-action", title: 'Erfahre mehr über OpenMensa' do
+            ' Erfahre mehr über OpenMensa
```

Comment:
You are free to make an alternative proposal. But normal links are in the current layout nearly invisible. The goal is that a viewer notice this three notable action from the text.